### DISCUSSION TITLE
Help to configure hash database

### DISCUSSION THREAD

**Discussion by msander2022** [sepinf-inc/IPED#879] [General]
Bom dia eu estou iniciando no mundo do excelente IPED  e estou com uma duvida de como configurar a importação da base NSRL via opção -importkff ,  é neste campo que altera   **kffDb = C:/IPED/IPED-KFF-1.2/kff.db**   se tiverem um modelo da configuração ja vai me ajudar ,  e aproveitando a oportunidade gostaria de saber como fazer esta configuração correta  neste parametro abaixo.

# Habilita a transcrição de áudios. A implementação padrão usa o Microsoft Azure. Pode-se chavear para Google Cloud.
# Ajuste a configuração em conf/AudioTranscriptConfig.txt e informe a chave na linha de comando via -XazureSubscriptionKey=KEY
enableAudioTranscription = false

**Comment by lfcnassif** [comment]
Have you read the user manual?
https://github.com/sepinf-inc/IPED/wiki/User-Manual

**Reply by msander2022** [reply]
Bom dia sim eu dei uma olhada com mais atenção e consegui resolver obrigado.
